sphinx_scality/layout.html

```diff
@@ -115,10 +115,13 @@ <h2><a href="{{ pathto(pagename) }}">{{ title }}</a></h2>
       <div class="sidebar-relinfo">
         {# TODO: add link to release notes and other hosted releases #}
         <p>Release: <span class="value">{{ release }}</span></p>
-      </div>
```

Comment:
You can't move this `div` outside the `if release` block, otherwise if you have `theme_releasenoteslink` set but not `release` (or the opposite), things will break.

Reply:
To help with this issue, I think you could instead write:

```html
      {%- block sidebarreleaseinfo %}
      <div class="sidebar-relinfo">
        {%- if release | default('') != '' %}
        <p>Release: <span class="value">{{ release }}</span></p>
        {%- endif %}

        {%- if theme_releasenoteslink | default('') != '' %}
        <a href="{{ theme_releasenoteslink }}">Release Notes</a>
        {%- endif %}
      </div>
      {%- endblock %} {# sidebarreleaseinfo #}
```